### PR TITLE
feat: new pages for identity

### PR DESCRIPTION
**Pull request by sammyluo**
## What/Why/How?

- new pages for identity

## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: identity/api-reference/error-codes.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## 401 Error: Unauthorized
+
+These errors are the second-most common and generally occur when either the user or the client lacks valid authentication credentials to perform.
+
+| Error Code            | Description                                                                                                                                                                                                                                                            |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| token_inactive       | Token is inactive because it is malformed, expired, or invalid. Token validation failed.                                                                                                                                                                      |
```
[the remark this answers is not in the document]
Comment (tunguyennnnn) [reply identity/api-reference/error-codes.md:45]: same thing with `request_unauthorized`

File: identity/guides/client-configurations.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+### Client Metadata
+
+The Client Metadata contains the client-specific information required to send requests to the Unstoppable authorization server. It is updated to reflect confirmed changes to client settings.
+
+```javascript
+{
+    clientID: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+    clientSecret: "xxxxxxxxxxxxxxxxxxxxxxxxxx",
+    clientAuthMethod: "client_secret_xxxx",
+    redirectUri: "http://localhost/",
```
Comment (tunguyennnnn) [inline identity/guides/client-configurations.md:53]: nit: will it work with routes only? I think the standard is `/login/callback` for other auth product

File: identity/guides/client-configurations.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+<figure>
+
+![Stripe account connected and revenue share configuration complete](/images/login-client-rev-share-complete.png '#width=65%')
+
+<figcaption>Revenue share configuration complete</figcaption>
+</figure>
+
+## Badges
+
+The **Badges** configuration page allows partners to submit custom badges that can be earned and displayed on users' Unstoppable Domains profile pages. For an example of a ud.me profile with multiple badges earned see [ud.me/sandy.nft](https://ud.me/sandy.nft).
```
Comment (tunguyennnnn) [inline identity/guides/client-configurations.md:137]:
```suggestion
The **Badges** configuration page allows partners to submit custom badges that can be earned and displayed on users' Unstoppable Domains profile pages. See [ud.me/sandy.nft](https://ud.me/sandy.nft) for an example of a ud.me profile with multiple badges earned.
```

File: identity/guides/faq.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+## Will Login be a paid service?
+
+No, Login with Unstoppable is a free feature members get with the purchase/ownership of their Web3 domain. It is free to use and free for apps to integrate.
+
+[comment]: # (How is this different from Facebook Login or Google Login?)
+
+## How is this different from Login with Ethereum?
+
+Login with Unstoppable is live and available for use. We see it as becoming the foundations of your digital identity. That’s why we don’t charge renewal fees on our domains, we don’t think you should ever be able to lose your identity.
```
Comment (tunguyennnnn) [inline identity/guides/faq.md:32]:
```suggestion
Login with Unstoppable is live and available for use. We see it as becoming the foundations of your digital identity. That’s why we don’t charge renewal fees on our domains, we don’t believe that you should ever be able to lose your identity.
```
I would just `believe` to stress UD missions

File: identity/guides/login-protocols/authorization-server.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
+---
+title: Login Authorization Server | Unstoppable Domains Developer Portal
```
Comment (tunguyennnnn) [inline identity/guides/login-protocols/authorization-server.md:2]: Screenshot 2023-03-23 at 7 18 48 AM
seems like this is not render properly

**Review by tunguyennnnn — COMMENTED** [review]
LGTM! Some minor comments, I'll approve after they are addressed

File: identity/api-reference/error-codes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sammyluo) [reply identity/api-reference/error-codes.md:45]: Not sure. It's the error that hydra returns

File: identity/guides/client-configurations.md (hunk 2 above)
Comment (sammyluo) [reply identity/guides/client-configurations.md:53]: It's the current implementation so I would leave the docs as is until the implementation is updated.

File: identity/guides/client-configurations.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+![Entering Redirect URIs](/images/login-enter-redirect-uris.gif '#width=70%')
+
+<figcaption>Add new Redirect URIs</figcaption>
+</figure>
+
+#### Rules for Redirect URIs
+
+The Redirect URIs follow three rules:
+
+1. `http` URIs must resolve to URIs hosted at `http://127.0.0.1` or `http://localhost`.
```
[the remark this answers is not in the document]
Comment (sammyluo) [reply identity/guides/client-configurations.md:81]: I would keep this content as is for now and discuss with @bcarl-ud for improvement

File: identity/guides/login-protocols/authorization-server.md (hunk 5 above)
Comment (sammyluo) [reply identity/guides/login-protocols/authorization-server.md:2]: Ah, it used tab over space 😅 Should be fixed now

**Review by tunguyennnnn — APPROVED** [review]

**Pull request merged**